modules/data.atmosphere/R/tdm_generate_subdaily_models.R

```diff
@@ -0,0 +1,226 @@
+##' Create statistical models to predict subdaily meteorology from daily means
```

Comment:
Probably fine for a first pass, but why do we assume the input data to be downscaled is daily? e.g. is there anything in this scheme that would categorically preclude extending the approach to downscaling 6hr reanalysis?